gpbft/gpbft.go

```diff
@@ -163,6 +181,8 @@ type instance struct {
 	// Decision state. Collects DECIDE messages until a decision can be made,
 	// independently of protocol phases/rounds.
 	decision *quorumState
+	// Instance data that all participants must agree on.
+	instanceData *InstanceData
```

Comment:
Please move this up, perhaps adjacent to proposal or value, which have similar properties.